requirements.txt

```diff
@@ -16,3 +16,4 @@ fonttools~=4.38.0
 ftfy~=6.1.1
 datasets~=2.11.0
 lpips~=0.1.4
+sentencepiece~=0.1.99
```

Comment:
I'm not sure that we need sentencepiece 🤔

Reply:
it is necessary library for NLLBtokenizer